front/app/containers/ProjectsShowPage/timeline/PhaseDescription.tsx

```diff
@@ -34,53 +27,54 @@ const Container = styled.div`
   `}
 `;
 
-const Header = styled.div<{ hasContent: boolean }>`
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
-  margin-bottom: ${(props) => (props.hasContent ? '30px' : '0px')};
-
-  ${isRtl`
-    flex-direction: row-reverse;
-  `}
-`;
-
 const StyledFileAttachments = styled(FileAttachments)`
   margin-top: 20px;
   max-width: 520px;
 `;
 
 interface Props {
-  projectId: string;
   phaseId: string | null;
+  phaseNumber: number;
   className?: string;
+  hidden: boolean;
 }
 
-const PhaseDescription = memo<Props>(({ projectId, phaseId, className }) => {
+const PhaseDescription = ({
+  phaseId,
+  phaseNumber,
+  className,
+  hidden,
+}: Props) => {
   const theme: any = useTheme();
-  const { windowWidth } = useWindowSize();
   const localize = useLocalize();
   const phase = usePhase(phaseId);
   const phaseFiles = useResourceFiles({
     resourceId: phaseId,
     resourceType: 'phase',
   });
 
-  const smallerThanSmallTablet = windowWidth <= viewportWidths.smallTablet;
   const content = !isNilOrError(phase)
     ? localize(phase.attributes.description_multiloc)
     : '';
   const contentIsEmpty =
     content === '' || content === '<p></p>' || content === '<p><br></p>';
-  const hasContent = !contentIsEmpty || !isEmpty(phaseFiles);
+  const descriptionHasContent = !contentIsEmpty || !isEmpty(phaseFiles);
 
   return (
-    <Container className={`e2e-phase-description ${className || ''}`}>
-      <Header hasContent={hasContent}>
-        <PhaseTitle projectId={projectId} selectedPhaseId={phaseId} />
-        {!smallerThanSmallTablet && <PhaseNavigation projectId={projectId} />}
-      </Header>
-      {!isNilOrError(phase) && hasContent && (
+    <Container
+      className={`e2e-phase-description ${className || ''}`}
```

Comment:
Seems there are some e2e tests for this feature. Can we check that they still pass after the changes?